app/mod_blog/tests.py

```diff
@@ -0,0 +1,139 @@
+import unittest
+from flask import Flask,url_for
+from .. import app,db,mod_user,mod_blog
+from ..mod_user.models import User,Blog
+from ..mod_blog.forms import BlogForm
+
+
+class BlogTestCase(unittest.TestCase):
+	@classmethod
+	def setUpClass(cls):
+		app.config.from_object('config.TestingConfig')
+		cls.app = app.test_client()
+		cls.app_context = app.test_request_context()                      
+		cls.app_context.push()       
+		db.create_all()
+
+	@classmethod 	
+	def tearDownClass(cls):
+		db.session.remove()
+		db.drop_all()
+
+	
+	def create_user(self):
+		user = User(username="test",email="test@test.com",password="As123456")
```

Comment:
@rizwans-cuelogic , Please use the Factory for test-data creation. That will help you in code re-usability. 
FYI : http://factoryboy.readthedocs.io/en/latest/